src/core_cli/Core_actions.ml

```diff
@@ -148,11 +148,11 @@ let dump_equivalences file =
 
 let dump_rule file =
   let file = Run_semgrep.replace_named_pipe_by_regular_file file in
-  let rules = Parse_rule.parse file in
+  let rules = Parse_rule.parse ~rewrite_rule_ids:None file in
   rules |> List.iter (fun r -> pr (Rule.show r))
 
 let prefilter_of_rules file =
-  let rules = Parse_rule.parse file in
+  let rules = Parse_rule.parse ~rewrite_rule_ids:None file in
```

Comment:
wouldn't it be better to make rewrite_rule_ids a default parameter like
?(rewrite_rule_ids=None), so you don't need to modify all those call sites?